src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -73,7 +74,7 @@ internal sealed class SolutionRestoreWorker : SolutionEventsListener, ISolutionR
         private Lazy<INuGetErrorList> _errorList;
         private readonly Lazy<IOutputConsoleProvider> _outputConsoleProvider;
 
-        private Lazy<INuGetExperimentationService> _nuGetExperimentationService;
+        private Lazy<INuGetFeatureFlagService> _nugetFeatureFlagService;
```

Comment:
Given that `_nugetFeatureFlagService` variable is initialized in the constructor, can we make this `readonly`?